packages/ag-charts-community/src/chart/data/dataService.ts

```diff
@@ -86,6 +86,7 @@ export class DataService<D extends object> extends Listeners<EventType, EventHan
     }
 
     private createThrottledFetch(requestThrottle: number) {
+        // eslint-disable-next-line sonarjs/no-misused-promises
```

Comment:
Why did it think this was misused?

Reply:
Actually let me tune the rule config for this down, so we remember to address it.